spec/controllers/api/v1/artists_controller_spec.rb

```diff
@@ -57,7 +62,18 @@
       expect(returned_json["name"]).to eq third_artist.name
       expect(returned_json["bio"]).to eq third_artist.bio
     end
-  end 
+  end
+  
+  describe "POST#create" do
```

Comment:
let's group these together so we don't repeat `POST#create` in multiple describe blocks - you can add `context` blocks to describe multiple similar things you want to test - you can nest them within each other
```ruby
describe "POST#create"
    context "artist is successfully added"
        it "creates the new artist"
        end
        
        it "returns json of the new artist"
        end
    end
    
    context "artist is unsuccessful"
    
    etc....
```